fdbserver/storageserver.actor.cpp

```diff
@@ -6679,8 +6817,17 @@ ACTOR Future<Void> storageServerCore(StorageServer* self, StorageServerInterface
 						if (self->db->get().logSystemConfig.recoveredAt.present()) {
 							self->poppedAllAfter = self->db->get().logSystemConfig.recoveredAt.get();
 						}
-						self->logCursor = self->logSystem->peekSingle(
-						    self->thisServerID, self->version.get() + 1, self->tag, self->storageTeamID, self->history);
+						if (!SERVER_KNOBS->ENABLE_PARTITIONED_TRANSACTIONS) {
```

Comment:
Nit: I typically do `if (SERVER_KNOBS->ENABLE_PARTITIONED_TRANSACTIONS) {` to avoid extra reasoning for `!` when reading the code.

Reply:
This is intended, so the old logic is in the first branch.

Reply:
I don't see it's necessary to keep old logic in the first branch and making code more readable to me is more desirable. Anyway, I don't have a strong opinion here.